File: Data Structures & Algorithms/top-k-elements-in-list/submission-10.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count={}
        freq=[[] for i in range(len(nums)+1)]
        res=[]
        for i in nums:
            count[i]=1+ count.get(i,0)
        
        for n,c in count.items():
            freq[c].append(n)
        for i in range(len(freq)-1,0,-1):
            for a in freq[i]:
                res.append(a)
                if len(res)==k:
                    return res

File: Data Structures & Algorithms/longest-common-prefix/submission-10.ts
class Solution {
    /**
     * @param {string[]} strs
     * @return {string}
     */
    longestCommonPrefix(strs: string[]): string {
        let res=""
        for (let i =0 ; i<strs[0].length;i++){
            for(let w=0; w<strs.length;w++){
                if(i===strs[w].length || strs[w][i]!==strs[0][i]){
                    return res
                }
            }
            res+=strs[0][i]
        }
        return res
    }
}
